roles/openshift_health_checker/openshift_checks/mixins.py

```diff
@@ -2,17 +2,14 @@
 from openshift_checks import get_var
 
 
-class NotContainerized(object):
+class NotContainerizedMixIn(object):
```

Comment:
Hmm, I had intently opted not for using the word "mixin" here. It will be referenced by folks writing checks, it affects readability.

IMHO this reads better

```python
class PackageAvailability(NotContainerized, OpenShiftCheck):
```

than

```python
class PackageAvailability(NotContainerizedMixIn, OpenShiftCheck):
```

I preferred to "pay the fee" in 1 file by having the pylint pragma, than having a worse name in N files.